server/users.js

```diff
@@ -0,0 +1,146 @@
+const express = require("express")
```

Comment:
Is it standard practice to separate the routes for different database components?

Reply:
(into different files)

Reply:
Yes, I would consider it to be a best practice for the sake of organization. For example, if you need to check an existing user route or add a new route, it's much easier (IMO) to open and edit the one file dedicated to that entity. Personally, I would much rather have 4 different files with 25 lines of code each as opposed to 1 file with 100 lines. It also helps when writing code in a team environment to not have different people trying to edit the exact same file at the same time.

Reply:
I was just listening to a podcast the other day and they were discussing the 1 vs many file approach. I'd have to re-listen to find the exact quote, but they called using a single file "scroll wheel driven development", "development by scroll wheel" or something along those lines. I think the only people who like the single file approach are the ones that love their vertically oriented monitors. :)